### DISCUSSION TITLE
Provenance texts with/without quote

### DISCUSSION THREAD

**Discussion by IRDnamik** [eblondel/geoflow#318] [Q&A] [answered]
The "Provenance" column in the metadata table causes errors if there is no quote. With relatively short texts the error is not encountered.

**Comment by eblondel** [comment]
can you paste here the text tha you try to use?

**Comment by IRDnamik** [comment]
Yes the text in "Provenance" field is : 

`statement:Les images ont été prises par un drone eBee, produit par senseFly. Les acquisitions se sont faites selon un plan de vol planifié par le logiciel eMotion, pour une hauteur maximale de 974 m et une couverture maximale de 10 km². Avec le logiciel Pix4Dmapper, les images acquises ont ensuite été géoréférencées puis assemblées. Enfin l'ensemble a été orthorectifié afin de former la mosaïques. La résolution spatiale maximale des images au sol est de 3 cm par pixel. `

Geoflow exit with : 
```
Error in if (is.na(fieldObj)) { : la condition est de longueur > 1            
De plus : Message d'avis :
Dans super$initialize(xml = xml, id = private$xmlElement, value = value,  :
  No ISO 'CI_RoleCode' codelist entry for value 'metadata'
```

**Comment by kikislater** [comment]
This was working before (could not state when) as well : 

```
statement:Drone : DJI Mavic 2 Pro

Hauteur de vol : 70m AGL 

Angle de la camera : 75°

Angle des transects : 0 - 180°

GSD (Ground Sampling Distance) résultant : 2cm/pixels

Logiciel utilisé pour le vol : Pix4D Capture

Décollage / Attérissage : Pont du bateau

Météo : Temps nuageux à couvert, légère brise se renforçant en fin de mission, marée remontante fort coefficient avec activité coralienne intense

EPI utilisé : casque et gants en cuir
```

**Comment by eblondel** [comment, marked as answer]
For long texts, paragraphs, or when you have several paragraphs with new lines, you have to quote the text, otherwise there is no other way for geoflow to know it is targeting the same property. Quoting a complex text is then whatvhas been introduced into geoflow to make it work.

**Comment by eblondel** [comment]
> This was working before (could not state when) as well : 
> 
> ```
> statement:Drone : DJI Mavic 2 Pro
> 
> Hauteur de vol : 70m AGL 
> 
> Angle de la camera : 75°
> 
> Angle des transects : 0 - 180°
> 
> GSD (Ground Sampling Distance) résultant : 2cm/pixels
> 
> Logiciel utilisé pour le vol : Pix4D Capture
> 
> Décollage / Attérissage : Pont du bateau
> 
> Météo : Temps nuageux à couvert, légère brise se renforçant en fin de mission, marée remontante fort coefficient avec activité coralienne intense
> 
> EPI utilisé : casque et gants en cuir
> ```

I dont know how this could have worked. For complex texts the quoting was introduced

**Comment by juliettefabre** [comment]
Hi, 
I just encountered the same problem with the Provenance column, though it used to work before, and though I have longer texts for other columns such as abstract. With quotes (") it works again.
